tests/integration/tier0/inhibit-if-kmods-is-not-supported/test_kmods_not_supported.py

```diff
@@ -35,14 +39,47 @@ def test_inhibit_if_custom_module_loaded(insert_custom_kmod, convert2rhel):
 
 def test_do_not_inhibit_if_module_is_not_loaded(shell, convert2rhel):
     assert shell("modprobe -r -v bonding").returncode == 0
+    # If custom module is not loaded the conversion is not inhibited.
     with convert2rhel(
-        ("-y --no-rpm-va --serverurl {} --username {} --password {} --pool {} --debug").format(
+        ("--no-rpm-va --serverurl {} --username {} --password {} --pool {} --debug").format(
             env.str("RHSM_SERVER_URL"),
             env.str("RHSM_USERNAME"),
             env.str("RHSM_PASSWORD"),
             env.str("RHSM_POOL"),
         )
     ) as c2r:
+        c2r.expect("Continue with the system conversion?")
+        c2r.sendline("y")
+        c2r.expect("Continue with the system conversion?")
+        c2r.sendline("y")
```

Comment:
Do we really need it to have these two lines here twice?

Reply:
I'm afraid so. If we cancel the conversion by sending CTRL+C the system will be left in a partially broken state. It's because the rollback function has problem with handling this SIGINT on OL7/C7 systems.  
More on the issue is mentioned here:
https://issues.redhat.com/browse/OAMG-5756
https://github.com/oamg/convert2rhel/pull/411

Reply:
ok, It seemed to me like some copy&paste error